Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -113,5 +136,43 @@ extension Message {
       }
     }
   }
+
+  /// Creates a new message by decoding the given `Data` containing a
+  /// serialized message in JSON format, interpreting the data as UTF-8 encoded
+  /// text.
+  ///
+  /// - Parameter jsonUTF8Data: The JSON-formatted data to decode, represented
+  ///   as UTF-8 encoded text.
+  /// - Parameter extensions: The extension map to use with this decode
+  /// - Parameter options: The JSONDecodingOptions to use.
+  /// - Throws: `JSONDecodingError` if decoding fails.
+  public init(
+    jsonUTF8Data: Data,
+    extensions: ExtensionMap?,
```

Comment:
Same about defaulting.

If keeping them, it might also make sense to make the other (order) inits call to these new ones so there is only one set of code to maintain.

Reply:
I've changed the older inits to delegate to the new ones to limit code duplication.

I'm still debating whether to actually remove the old inits or not.  I'll leave the other conversation open to track this while I cogitate.  ;-)